charts/lab-partner/templates/secret.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: lab-partner
+type: Opaque
+data:
```

Comment:
`data` is meant to take base64 encoded strings as values.  you can use `stringData` here instead so you don't have to do that.